src/components/app-rating-mobile/AppRatingMobile.tsx

```diff
@@ -9,18 +9,41 @@ import { styles } from '~/components/app-rating-mobile/AppRatingMobile.styles'
 
 interface AppRatingMobileProps extends RatingProps {
   reviewsCount: number
+  linkHash?: ReactElement
```

Comment:
it should not be here. you should pass prop 'to' for link

Reply:
Fixed.